lib/takeaway.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'menu'
+require_relative 'order'
+require_relative 'bill'
+require_relative 'sms_confirm'
+
+class Takeaway
+
+  attr_reader :menu, :order, :bill, :sms
+
+  def initialize(menu = Menu, order = Order.new, bill = Bill.new, sms = SMS.new)
+    @menu = menu
+    @order = order
+    @bill = bill
+    @sms = sms
+  end
+
+  def show_menu
```

Comment:
Should this be menu.show, then in your initializer should it state menu = Menu.new